src/server/api/routes/api-router.js

```diff
@@ -3,8 +3,8 @@ const express = require('express');
 const router = express.Router();
 
 // Router imports
-const modulesRouter = require('./modules.router');
 const channelsRouter = require('./channels-router');
+const modulesRouter = require('./modules.router');
```

Comment:
You should undo changes in this file.

Reply:
I do not got it. which ones should I remove?

Reply:
sorry, I get it.